docs/source/deployment.rst

```diff
@@ -8,43 +8,91 @@ This will be run automatically when you attempt to commit code but if you want t
 
 .. code-block:: console
 
-    poetry run pre-commit run --all-files
+    $ poetry run pre-commit run --all-files
 
 This app is set up to use circleci, but could be extended to any build process. Circle will automatically run:
 
 .. code-block:: console
 
-    poetry run pre-commit run --all-files
-    poetry run playwright install
-    poetry run coverage run --source='.' -m pytest
+    $ poetry run pre-commit run --all-files
```

Comment:
Why add the dollar signs? This only convention for some shell terminals, but not all.

Reply:
Bash is a project prerequisite as it currently stands, so I'm assuming a bash-like shell. If you feel like it's detracting from the content we can remove it, but the character itself isn't selectable when copying the commands from the generated docs.